src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1519,6 +1520,12 @@ def generate(
                 find all the possible language tokens in the `model.generation_config.lang_to_id` dictionary.
             is_multilingual (`bool`, *optional*):
                 Whether or not the model is multilingual.
+            initial_prompt_ids (`List[int]`, *optional*):
+                Optional list of token IDs created by passing text to `processor.create_initial_prompt_ids` that is
+                provided as a prompt for the first window. This can be used to provide or "prompt-engineer" a context
```

Comment:
What kind of window does this refer to? (I'm assuming what's called window here is what we call a chunk. If that's the case we should be consistent with the terminology.)

Reply:
The "first window" refers to the initial segment of the input speech, so I believe it can be used interchangeably with chunk yes. Updated the wording to use 'chunk'